internal/services/network/network_manager_management_group_connection_resource_test.go

```diff
@@ -30,7 +30,7 @@ func testAccNetworkManagerManagementGroupConnection_basic(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("connection_state"),
```

Comment:
Can we add a comment above this with a short explanation as to why we're ignoring this on import

Reply:
added